cpp/examples/README.md

```diff
@@ -0,0 +1,32 @@
+# libcugraph examples
+
+Example codes on how to use libcugraph.
+
+## Contents
+
+- single_gpu: simple code on how to use libcugraph to run different graph algorithms in a single GPU node.
+
+- multi_gpu: example code on how to use libcugraph to run different graph algorithms in a multi GPU node.
+
+-  graph_partitioning: contains code to explain vertex and edge partitioning in cugraph.
```

Comment:
What's the purpose of this? Users don't need to understand partitioning to use libcugraph.

Reply:
For application developer who wants to contribute and for those who want to implement their own/custom algorithms would like to understand how partitioning work.